sdk/core/src/main/java/com/klaviyo/coresdk/networking/requests/IdentifyApiRequest.kt

```diff
@@ -0,0 +1,30 @@
+package com.klaviyo.coresdk.networking.requests
+
+import android.util.Base64
+import com.klaviyo.coresdk.Registry
+import com.klaviyo.coresdk.model.Profile
+import org.json.JSONObject
+
+/**
+ * Defines information unique to building a valid identify request
+ *
+ * @param profile Profile attributes to send
+ */
+internal class IdentifyApiRequest(profile: Profile) : KlaviyoApiRequest(
+    "api/identify",
+    RequestMethod.GET
+) {
+
+    override var query: Map<String, String> = profile.let { properties ->
+        val data = JSONObject(
+            mapOf(
+                "token" to Registry.config.apiKey,
+                "properties" to JSONObject(properties.toMap())
+            )
+        ).toString()
+
+        mapOf(
+            "data" to Base64.encodeToString(data.toByteArray(), Base64.NO_WRAP)
```

Comment:
Yah if you make it a post you don't this base64 stuff anymore.

Reply:
Or I guess just use v3.

Reply:
Thanks, I'll put a comment in the V3 ticket to both update profile call to v3 but also change the legacy call to use POST.